plugin-gradle/src/main/java/com/diffplug/gradle/spotless/JsonExtension.java

```diff
@@ -71,6 +75,14 @@ public RomeJson rome(String version) {
 		return romeConfig;
 	}
 
+	public ApplyJsonPatchConfig applyJsonPatch(List<Map<String, Object>> patch) {
+		return new ApplyJsonPatchConfig(patch);
+	}
+
+	public ApplyJsonPatchConfig applyJsonPatch(String zjsonPatchVersion, List<Map<String, Object>> patch) {
```

Comment:
For now, I don't understand how this works, the maven API of just passing a string seems more straight-forward to me. Updating the docs will help.

Reply:
I've updated the [docs](https://github.com/diffplug/spotless/pull/1753/files#diff-57c3e8c6144dec2a58860a00ac656d3372ffc4a72d334d3e3364ff1d72490150R876) now which hopefully explain this signature a little better.